test/e2e/util.go

```diff
@@ -993,6 +995,22 @@ func Diff(oldPods []*api.Pod, curPods []*api.Pod) PodDiff {
 // namespace lifecycle for handling cleanup).
 func RunRC(config RCConfig) error {
 
+	// Prometheus stuff: Setup metrics.
+	runningMetric := prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
the push is orthagonal to the gauge, so i could if you want me to, but not sure its needed